pkg/reconciler/v1alpha1/channel/channel_test.go

```diff
@@ -33,6 +31,7 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
```

Comment:
I find VS Code always (or very often) changing these imports, while Goland does not.

Should this be - somehow - configurable, as a "coding standard" ? 

Often these changes cause noise

Reply:
My observation is that it will only alphabetize lists of imports that don't have any new lines between them. So, by removing the new line from the old line 26, GoLand alphabetized the whole section.

As to why GoLand throws seemingly random new lines into the import section, I have no idea.

In the end GoLand (at least for me) is setup to not do any formatting itself, rather it calls `goimports`.